packages/esm-generic-patient-widgets-app/src/index.ts

```diff
@@ -25,6 +30,7 @@ function setupOpenMRS() {
         meta: {
           columnSpan: 4,
         },
+        offline: { showRecordVitalsButton: true },
```

Comment:
I haven't deeply looked into this, but @brandones is most likely right here, the name `showRecordVitalsButton` seems to belong inside the vitals widget. If so, it can be removed from here (and the declaration can just be replaced with `offline: true` instead).

Reply:
@brandones @manuelroemer you're right, this showRecordVitalsButton probably was some copy paste gone wrong. It seemed to work anyway though as offline != false
I've now replaced with `offline: true`